Source/Core/Common/CMakeLists.txt

```diff
@@ -65,4 +65,18 @@ if(NOT APPLE AND NOT ANDROID)
 	set(LIBS ${LIBS} rt)
 endif()
 
+if(VTUNE)
```

Comment:
Yeah, it would probably make sense but it's only used here and thus reduces recompile times. Do you want me to move it?